crates/lang/macro/Cargo.toml

```diff
@@ -28,6 +28,7 @@ ink_metadata = { version = "3.0.0-rc3", path = "../../metadata/" }
 ink_env = { version = "3.0.0-rc3", path = "../../env/" }
 ink_storage = { version = "3.0.0-rc3", path = "../../storage/" }
 ink_lang = { version = "3.0.0-rc3", path = ".." }
+ink_prelude = { version = "3.0.0-rc3", path = "../../prelude/" }
```

Comment:
does `dev-dependency` work here instead?

Reply:
It is already under `[dev-dependencies]`, did you mean that?

Reply:
nvm :facepalm:  I didnt see that in the past view